packages/pmml-editor/it-tests/integration/scorecard-model.ts

```diff
@@ -0,0 +1,491 @@
+
+    cy.ouiaType("model-setup-overview").click();
+    cy.ouiaType("edit-model-setup").should("be.visible");
+    cy.ouiaId("baseline-score").clear();
+
+    cy.ouiaType("filler").first().click();
+
+    cy.ouiaType("model-setup-overview")
+      .find("[data-ouia-component-type=model-property]")
+      .should(($label) => {
+        expect($label).to.have.length(5);
+        expect($label[0]).to.have.text("Is Scorable:\u00A0Yes");
+        expect($label[1]).to.have.text("Function:\u00A0regression");
+        expect($label[2]).to.have.text("Use Reason Codes:\u00A0Yes");
+        expect($label[3]).to.have.text("Reason Code Algorithm:\u00A0pointsBelow");
+        expect($label[4]).to.have.text("Baseline Method:\u00A0other");
+      });
+
+    cy.ouiaType("model-setup-overview")
+      .find("[data-ouia-component-type=invalid-label]")
+      .should(($label) => {
+        expect($label).to.have.length(1);
+        expect($label[0]).to.have.text("Baseline Score:\u00A0Missing");
+      });
+  });
+
+  it("Attribute 'Algorithm' - set and unset", () => {
+    cy.ouiaType("model-setup-overview").click();
+    cy.ouiaType("edit-model-setup").should("be.visible");
+    cy.ouiaId("algorithm").type("Test something");
+
+    cy.ouiaType("filler").first().click();
+
+    cy.ouiaType("model-setup-overview")
+      .find("[data-ouia-component-type=model-property]")
+      .should(($label) => {
+        expect($label).to.have.length(6);
+        expect($label[0]).to.have.text("Is Scorable:\u00A0Yes");
+        expect($label[1]).to.have.text("Function:\u00A0regression");
+        expect($label[2]).to.have.text("Algorithm:\u00A0Test something");
+        expect($label[3]).to.have.text("Use Reason Codes:\u00A0Yes");
+        expect($label[4]).to.have.text("Reason Code Algorithm:\u00A0pointsBelow");
+        expect($label[5]).to.have.text("Baseline Method:\u00A0other");
+      });
+
+    cy.ouiaType("model-setup-overview")
+      .find("[data-ouia-component-type=invalid-label]")
+      .should(($label) => {
+        expect($label).to.have.length(1);
+        expect($label[0]).to.have.text("Baseline Score:\u00A0Missing");
+      });
+
+    cy.ouiaType("model-setup-overview").click();
+    cy.ouiaType("edit-model-setup").should("be.visible");
+    cy.ouiaId("algorithm").clear();
+
+    cy.ouiaType("filler").first().click();
+
+    cy.ouiaType("model-setup-overview")
+      .find("[data-ouia-component-type=model-property]")
+      .should(($label) => {
+        expect($label).to.have.length(6);
+        expect($label[0]).to.have.text("Is Scorable:\u00A0Yes");
+        expect($label[1]).to.have.text("Function:\u00A0regression");
+        /*
+         * This attribute is not required by PMML definition.
+         * If its value is the empty string then the particular label should be removed.
+         * TODO: reach an agreement about functionality.
+         */
+        expect($label[2]).to.have.text("Algorithm:\u00A0");
```

Comment:
@manstis You have better knowledge of code. Please create new JIRA where you fix it. There is suggestion how you could fix the "Attribute 'Algorithm' - set and unset" test when new behaviour will be implemented:
```suggestion
        expect($label).to.have.length(5);
        expect($label[0]).to.have.text("Is Scorable:\u00A0Yes");
        expect($label[1]).to.have.text("Function:\u00A0regression");
        expect($label[2]).to.have.text("Use Reason Codes:\u00A0Yes");
        expect($label[3]).to.have.text("Reason Code Algorithm:\u00A0pointsBelow");
        expect($label[4]).to.have.text("Baseline Method:\u00A0other");
```

Reply:
OK, see https://issues.redhat.com/browse/FAI-558. I'll work on it next week.